src/studio/src/designer/DataModeling/Converter/Json/Strategy/JsonSchemaGeneralConverter.cs

```diff
@@ -462,7 +562,9 @@ private void HandleSimpleTypeRestriction(XmlSchemaSimpleType simpleType, WorkLis
             simpleType.Content = restriction;
 
             // the final builtin base type for this simple type refinement chain
-            var targetBaseType = XmlQualifiedName.Empty;
+#pragma warning disable S1854 // Unused assignments should be removed
```

Comment:
Is it a false positive?

Reply:
Yes, I think so. If you look at usage below, I think there are branches of the if blocks that don't set this. But good with another pair of eyes.